shared/constants/fs.tsx

```diff
@@ -944,9 +944,9 @@ export const getMainBannerType = (
   overallSyncStatus: Types.OverallSyncStatus
 ): Types.MainBannerType =>
   kbfsDaemonStatus.onlineStatus === Types.KbfsDaemonOnlineStatus.Offline
-    ? flags.kbfsOfflineMode
-      ? Types.MainBannerType.Offline
-      : Types.MainBannerType.None
+    ? Types.MainBannerType.Offline
```

Comment:
this pattern is super hard for me to read when there are so many ?. Can we switch to a series of else ifs here?